src/material/Enchantment.ts

```diff
@@ -3,7 +3,47 @@ import { ToJava } from "../runtime/ToJava";
+    public static readonly BINDING_CURSE: Enchantment = JAVA_ENCHANTMENT.BINDING_CURSE;
+    public static readonly CHANNELING: Enchantment = JAVA_ENCHANTMENT.CHANNELING;
+    public static readonly DAMAGE_ALL: Enchantment = JAVA_ENCHANTMENT.DAMAGE_ALL;
+    public static readonly DAMAGE_ARTHROPODS: Enchantment = JAVA_ENCHANTMENT.DAMAGE_ARTHROPODS;
+    public static readonly DAMAGE_UNDEAD: Enchantment = JAVA_ENCHANTMENT.DAMAGE_UNDEAD;
+    public static readonly DEPTH_STRIDER: Enchantment = JAVA_ENCHANTMENT.DEPTH_STRIDER;
+    public static readonly DIG_SPEED: Enchantment = JAVA_ENCHANTMENT.DIG_SPEED;
+    public static readonly DURABILITY: Enchantment = JAVA_ENCHANTMENT.DURABILITY;
+    public static readonly FIRE_ASPECT: Enchantment = JAVA_ENCHANTMENT.FIRE_ASPECT;
+    public static readonly FROST_WALKER: Enchantment = JAVA_ENCHANTMENT.FROST_WALKER;
+    public static readonly IMPALING: Enchantment = JAVA_ENCHANTMENT.IMPALING;
+    public static readonly KNOCKBACK: Enchantment = JAVA_ENCHANTMENT.KNOCKBACK;
+    public static readonly LOOT_BONUS_BLOCKS: Enchantment = JAVA_ENCHANTMENT.LOOT_BONUS_BLOCKS;
+    public static readonly LOOT_BONUS_MOBS: Enchantment = JAVA_ENCHANTMENT.LOOT_BONUS_MOBS;
+    public static readonly LOYALTY: Enchantment = JAVA_ENCHANTMENT.LOYALTY;
+    public static readonly LUCK: Enchantment = JAVA_ENCHANTMENT.LUCK;
+    public static readonly LURE: Enchantment = JAVA_ENCHANTMENT.LURE;
+    public static readonly MENDING: Enchantment = JAVA_ENCHANTMENT.MENDING;
+    public static readonly MULTISHOT: Enchantment = JAVA_ENCHANTMENT.MULTISHOT;
+    public static readonly OXYGEN: Enchantment = JAVA_ENCHANTMENT.OXYGEN;
+    public static readonly PIERCING: Enchantment = JAVA_ENCHANTMENT.PIERCING;
+    public static readonly PROTECTION_ENVIRONMENTAL: Enchantment = JAVA_ENCHANTMENT.PROTECTION_ENVIRONMENTAL;
+    public static readonly PROTECTION_EXPLOSIONS: Enchantment = JAVA_ENCHANTMENT.PROTECTION_EXPLOSIONS;
+    public static readonly PROTECTION_FALL: Enchantment = JAVA_ENCHANTMENT.PROTECTION_FALL;
+    public static readonly PROTECTION_FIRE: Enchantment = JAVA_ENCHANTMENT.PROTECTION_FIRE;
+    public static readonly PROTECTION_PROJECTILE: Enchantment = JAVA_ENCHANTMENT.PROTECTION_PROJECTILE;
+    public static readonly QUICK_CHARGE: Enchantment = JAVA_ENCHANTMENT.QUICK_CHARGE;
+    public static readonly RIPTIDE: Enchantment = JAVA_ENCHANTMENT.RIPTIDE;
+    public static readonly SILK_TOUCH: Enchantment = JAVA_ENCHANTMENT.SILK_TOUCH;
+    public static readonly SOUL_SPEED: Enchantment = JAVA_ENCHANTMENT.SOUL_SPEED;
+    public static readonly SWEEPING_EDGE: Enchantment = JAVA_ENCHANTMENT.SWEEPING_EDGE;
+    public static readonly THORNS: Enchantment = JAVA_ENCHANTMENT.THORNS;
+    public static readonly VANISHING_CURSE: Enchantment = JAVA_ENCHANTMENT.VANISHING_CURSE;
+    public static readonly WATER_WORKER: Enchantment = JAVA_ENCHANTMENT.WATER_WORKER;
```

Comment:
@connerdouglass I was just experimenting with this, but not sure if I like it, thoughts?

I'm thinking it should return a JS instance of Enchantment instead

Reply:
I agree with you, I think it should return a JS instance. Ideally with the customrealms/core library we don't want to expose any Java values that aren't wrapped

Reply:
Hopefully updating this file is quick with multiple cursors!